### PR TITLE
Traffic Light Scenarios

### PR DESCRIPTION
**Pull request by mxgrey**
This PR provides the testing scenarios for the traffic light fleet adapter introduced in https://github.com/osrf/rmf_core/pull/226.

### CODE REVIEW

**Review by Yadunund — CHANGES_REQUESTED** [review]
All the scenarios work great!
Before we merge this in, could we modify `battle_royale.launch.xml`, `triple_H.launch.xml` and `office_mock_traffic_light.launch.xml` to also support launching with `ignition`? We can follow the template in the latest [`master`](https://github.com/osrf/rmf_demos/blob/de780c181c36261b80025bbf8d50713f1367f68e/demos/launch/office.launch.xml#L14-L61)

**Comment by mxgrey** [comment]
I felt there was a lot of code duplication between the different launch files, so I've refactored how the simulations get launched across all the different maps.

I don't have ignition installed, so we should definitely have an ignition user test out whether these changes still work for launching ignition.

File: demos/launch/triple_H.launch.xml [hunk 1]
```diff
@@ -9,27 +11,12 @@
...
-      <executable cmd="gzclient --verbose $(var world_path)" output="both">
-        <env name="GAZEBO_MODEL_PATH" value="$(var model_path)" />
-        <env name="GAZEBO_RESOURCE_PATH" value="$(var resource_path)" />
-        <env name="GAZEBO_PLUGIN_PATH" value="$(var plugin_path)" />
-      </executable>
-    </group>
-  </group>
+  <!-- Simulation launch -->
+  <include file="$(find-pkg-share demos)/simulation.launch.xml">
+    <arg name="map_name" value="office" />
```
Comment (Yadunund) [inline demos/launch/triple_H.launch.xml:16]: `value` should equal `triple_H` here

Comment (mxgrey) [reply demos/launch/triple_H.launch.xml]: Fixed!

**Review by Yadunund — APPROVED** [review]
Thank you for cleaning up the launch files! Ignition and gazebo simulations work as expected :+1:

**Pull request merged**